### PR TITLE
Update ongoing maintenance list

### PR DESCRIPTION
**Pull request by ccostino**
This update adds a step to review all expiring certificates to make sure we do not miss any coming up in the near future.

Note that this is a temporary step until alerts are back in place for certificate expirations.

## Changes proposed in this pull request:
- Cleans up formatting at the top of the guide a bit
- Adds a small section for checking expiring certificates

:sunglasses: [PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/ongoing-platform-maintenance-updates/docs/ops/maintenance-list/)

## Security Considerations
- Low/no risk as it is just reformatting information already on the page and noting another shell command to run.

### CODE REVIEW

**Comment by ccostino** [comment]
I wasn't sure how best to capture these instructions and if these apply across all environments or just one or two of them, so please provide any feedback/improvements!

File: _docs/ops/maintenance-list.md [hunk 1]
```diff
@@ -66,6 +66,10 @@ Review the [detailed guide on customer support]({{site.baseurl}}/docs/ops/custom
 - In [logs.fr.cloud.gov](https://logs.fr.cloud.gov/), go under "Management" -> "Advanced Settings" and check the Kibana [timezone setting](https://www.elastic.co/guide/en/kibana/current/advanced-options.html) (`dateFormat:tz`) - it should be set to `Browser`. If anyone has changed it, change it back to `Browser`.
 - Check for any new CVEs in [Elasticsearch](https://www.cvedetails.com/vulnerability-list/vendor_id-13554/Elasticsearch.html), and [Redis](https://www.cvedetails.com/vulnerability-list/vendor_id-15183/product_id-31837/Pivotal-Software-Redis.html).
 
+## Review expiring certificates
+
+From the jumpbox in each of our four environments, `tooling`, `development`, `staging`, and `production`, run `doomsday dashboard` to review all expiring certificates. Make note of any that are expiring within two weeks and make sure the team is aware.
+
```
Comment (ChrisMcGowan) [inline _docs/ops/maintenance-list.md:75]: You can also look at the GUI while on VPN: https://doomsday.fr.cloud.gov/

Comment (ccostino) [reply _docs/ops/maintenance-list.md:75]: Ah, that's much easier, thank you!  I also see what you mean about Comic Sans... 😆

**Comment by ccostino** [comment]
I added a couple of notes to call out when the VPN is needed and refined the instructions for checking expired certificates.

Does this process make sense to everyone?  Is there another way we should be doing this?

**Review by bengerman13 — APPROVED** [review]

**Pull request merged**